iroh-net/src/magicsock.rs

```diff
@@ -3085,48 +3085,48 @@ pub(crate) mod tests {
 
     #[tokio::test(flavor = "multi_thread")]
     async fn test_two_devices_setup_teardown() -> Result<()> {
-        setup_logging();
-
-        let devices = Devices {
-            stun_ip: "127.0.0.1".parse()?,
-        };
-
-        for _ in 0..10 {
-            let (derp_map, _, cleanup) = run_derp_and_stun(devices.stun_ip).await?;
-            println!("setting up magic stack");
-            let m1 = MagicStack::new(derp_map.clone()).await?;
-            let m2 = MagicStack::new(derp_map.clone()).await?;
-
-            let cleanup_mesh = mesh_stacks(vec![m1.clone(), m2.clone()]).await?;
+        crate::test_utils::with_logging(async move {
```

Comment:
All the test code here is untouched, only wrapped in this function call and async block.